uniffi_bindgen/src/bindings/swift/gen_swift/callback_interface.rs

```diff
@@ -2,8 +2,6 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
I would _like_ to comment on the commit message .... but GitHub doesn't allow that.

Anyway, typo fixes in the second commit:

```patch
- This types implemen the `FfiConverter` protocol, which replaces
+ These types implement the `FfiConverter` protocol, which replaces
```

and 

```patch
- code, which allows deleting a lot of lines from the gen_python modules.
+ code, which allows deleting a lot of lines from the gen_swift modules.
```